flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/SimpleDataUtil.java

```diff
@@ -284,11 +284,7 @@ public static void assertRecordsEqual(List<Record> expected, List<Record> actual
   public static void assertTableRecords(Table table, List<Record> expected, Duration timeout) {
     Awaitility.await("expected list of records should be produced")
         .atMost(timeout)
-        .untilAsserted(
-            () -> {
-              equalsRecords(expected, tableRecords(table), table.schema());
```

Comment:
This does not do anything. The method is just an "equals" implementation, but the result is not used anywhere.
So I removed the call.